va/va_enc_hevc.h

```diff
@@ -747,8 +780,23 @@ typedef struct _VAEncPictureParameterBufferHEVC {
         } bits;
         uint16_t value;
     } scc_fields;
+
+    /** \brief This Value indicates the source down scaling ratio for look
+     *  ahead pass (valid only when lookahead_phase == 1).
+     */
+    union {
+        struct {
+            /** \brief horizontal scaling ratio = (x16_minus1_x + 1) / 16 */
+            uint8_t    x16_minus1_x : 4;
```

Comment:
need to figure out the supported ratio

Reply:
1, 2, 4, 8

Reply:
x and y scaling ratio should be the same?

Reply:
Capabiltiy in VAConfigAttribValLookAhead for ds ratio added.